torch/testing/_internal/common_modules.py

```diff
@@ -211,6 +211,7 @@ def __init__(self,
                  skips=(),  # Indicates which tests to skip
                  decorators=None,  # Additional decorators to apply to generated tests
                  dtypes=floating_types(),  # dtypes this function is expected to work with
+                 dtypesIfMPS=(torch.float32,),  # dtypes this function is expected to work with on MPS
```

Comment:
Is this necessary if it's always torch.float32?

Reply:
Hm I was thinking that if MPS adds support for other dtypes then this kwarg would be useful

Reply:
MPS supports float16, so please add it here